packages/wallet-ui/src/components/ui/organism/NoFlaskModal/NoFlaskModal.style.ts

```diff
@@ -1,6 +1,6 @@
 import styled from 'styled-components';
```

Comment:
We still want to use snaps with Flask ? if it's not the case you can add a comment to remember that we need to remove this component after releasing the snaps in production

Reply:
noted, will add a comment to remove this component after release